docs/quickstarts/integrations-slack-notifs-qs/integrations-slack-notifs-qs.yml

```diff
@@ -61,21 +61,25 @@ spec:
 
         3. Create an incoming webhook:
 
-          a. Under **Settings** in the left navigation, click **Basic Information**.
+          a. Under **Settings** in the left navigation, click **Install App**.
 
-          b. In the **Install your app** section, click **Install to Workspace**.
+          b. In the **Install App** section, click **Install to _workspace_**.
 
-          c. Select a channel or user for your Slack app to post notifications and click **Allow**.
+          c. Select a channel or user for your Slack app to send notifications to and click **Allow**.
 
         4. (Optional) Configure how your console notifications display in Slack:
 
-          a. Scroll down to **Display Information** and configure your app description, icon, and background color.
+          a. Under **Settings** in the left navigation, click **Basic Information**.
+          
+          b. Scroll down to **Display Information** and configure your app name, description, icon, and background color.
```

Comment:
configure or:
enter
specify

Reply:
I'm going to change it to 'edit', only because the App name is already there when you go to this section. I think 'edit' works for the other items too.